WalletWasabi.Fluent/Views/Wallets/Home/History/Details/TransactionDetailsView.axaml

```diff
@@ -35,6 +35,17 @@
 
       <Separator />
 
+      <!-- Destination Address -->
+      <c:PreviewItem Icon="{StaticResource person_regular}"
```

Comment:
In send we have different icon used:

Send:
![image](https://github.com/zkSNACKs/WalletWasabi/assets/2297442/09b8c6f8-f135-49c0-a34c-7a40ae988948)
PR:
![image](https://github.com/zkSNACKs/WalletWasabi/assets/2297442/1ace4a61-e080-4893-b220-42d38deee1dc)

Reply:
Also it uses different naming, not sure about that

Reply:
Using it now.